pkg/cmd/kafka/openshift-cluster/openshiftclustercmdutil/dedicated_util.go

```diff
@@ -42,15 +42,18 @@ func (v *Validator) ValidatorForMachinePoolNodes(val interface{}) error {
 	return nil
 }
 
-func CreatePromptOptionsFromClusters(clusterList *kafkamgmtclient.EnterpriseClusterList, clusterMap *map[string]v1.Cluster) *[]string {
+func CreatePromptOptionsFromClusters(clusterList *kafkamgmtclient.EnterpriseClusterList, clusterMap *map[string]v1.Cluster) (*[]string, error) {
 	promptOptions := []string{}
 	validatedClusters := ValidateClusters(clusterList)
+	if len(validatedClusters.Items) == 0 {
+		return &promptOptions, fmt.Errorf("no registered OpenShift clusters found, an OpenShit cluster must be registered to use the service ")
```

Comment:
nit: Will it be tough to use the localizer here?

Reply:
great catch. I've updated it to use localizer 👍